### DISCUSSION TITLE
更新订阅提示 SequelizeUniqueConstraintError Validation error

### DISCUSSION THREAD

**Discussion by xixi-furry** [whyour/qinglong#1825] [Q&A]
-> 添加失败(SequelizeUniqueConstraintError Validation error)

**Comment by WinterBoring** [comment]
同样，以前好像没有这个问题

**Comment by YKYCYKYC** [comment]
我也是这个报错，是在更新/添加环境变量时出现的，今天发现的，我的版本是2.15.11，不知道什么时候开始有的，然后我升级了2.15.12，还是有这个问题

**Reply by YKYCYKYC** [reply]
楼上两位问题解决了吗，是需要修改什么配置文件吗，有没有大佬能指点一下，谢谢

**Reply by whyour** [reply]
> 我也是这个报错，是在更新/添加环境变量时出现的，今天发现的，我的版本是2.15.11，不知道什么时候开始有的，然后我升级了2.15.12，还是有这个问题

环境变量重复会提示这个

**Comment by lanwangchuan** [comment]
订阅重复了吧，我的是这样，不确定你的具体情况，可以试试

**Comment by YKYCYKYC** [comment]
确实是重复了，感谢大佬指点

在 2023年7月26日星期三，LingYi ***@***.***> 写道：
